src/main/java/cubicchunks/network/ClientHandler.java

```diff
@@ -81,23 +79,24 @@ public void handle(PacketCube packet) {
 		int cubeY = getY(cubeAddress);
 		int cubeZ = getZ(cubeAddress);
 
-		Column column = cubeCache.getColumn(cubeX, cubeZ);
+		Column column = cubeCache.provideChunk(cubeX, cubeZ);
```

Comment:
How does it disable it? ChunkProviderClient.provideChunk() will provide blackChunk (BlankColumn) when there is no chunk loaded. So column.isEmpty() still works.